package.json

```diff
@@ -47,6 +47,7 @@
     "less": "^3.0.1",
     "less-loader": "^4.0.5",
     "react-redux": "^5.0.7",
+    "styled-components": "^3.2.1",
```

Comment:
Denne har ganske stor påvirkning på hvordan kodebasen kommer til å utvikle seg, og bør tenkes godt igjennom imho. 
Hvordan blir interop med less etc?

Reply:
Mmm, mye som må avklares her, men vi har bestemt oss for å teste ut nå i en tidlig fase, og det er kanskje den enkleste måten å finne ut av hvordan det blir i bruk. Men ser at pakka har over 50k downloads om dagen og et veldig aktivt utvikler-team, så regner med at mange av de mer "synlige" problemstillingene har blitt eller raskt blir adressert. Men viser det seg å ikke funke får vi bare rulle tilbake. Siden styled-components bruker vanlig css-syntaks vil det ikke bli en veldig stor jobb.